internal/resume/resume.go

```diff
@@ -7,6 +7,7 @@ import (
 )
 
 type Resume struct {
+	CensoringEnabled bool             `yaml:"censoringenabled"`
```

Comment:
This should be checked in `resume_test.go`

Reply:
And the yaml variable should follow camelCase, so `censoringEnabled`

Reply:
Done